pkg/trustedresources/verifier/verifier.go

```diff
@@ -29,10 +29,9 @@ import (
 	"github.com/sigstore/sigstore/pkg/signature/kms"
 
 	// TODO(#5976): consider move these registration to cmd/controller/main.go
-	_ "github.com/sigstore/sigstore/pkg/signature/kms/aws"        // imported to execute init function to register aws kms
-	_ "github.com/sigstore/sigstore/pkg/signature/kms/azure"      // imported to execute init function to register azure kms
-	_ "github.com/sigstore/sigstore/pkg/signature/kms/gcp"        // imported to execute init function to register gcp kms
-	_ "github.com/sigstore/sigstore/pkg/signature/kms/hashivault" // imported to execute init function to register hashivault kms
```

Comment:
Curious why there seems no other functions related with the removal?

Reply:
yes because these are used to call init() functions to register those vendors

Reply:
By init() you mean the ones in ./test?

Reply:
no, this one https://github.com/tektoncd/pipeline/blob/03215a3681a55202c899d411a15cc391e1ed38de/vendor/github.com/sigstore/sigstore/pkg/signature/kms/gcp/client.go#L44-L48

Reply:
Okay! Could you update in the comment/ commit message to specify in which way this PR is going to unblock the development by clarifying what's the long term goal also?

Reply:
https://go.dev/doc/effective_go#blank_import

The init functions in those packages register the specific package

Reply:
> Okay! Could you update in the comment/ commit message to specify in which way this PR is going to unblock the development by clarifying what's the long term goal also?

sure! Thanks for the suggestion!